packages/main/src/Input.js

```diff
@@ -720,18 +726,18 @@ class Input extends UI5Element {
 	}
 
 	async openPopover() {
-		this.popover = await this._getPopover();
-		if (this.popover) {
+		const popover = await this._getPopover();
+
+		if (popover) {
 			this._isPopoverOpen = true;
-			this.popover.openBy(this);
```

Comment:
I'm fine by deleting `this.popover`, are you sure no other component's using it?